diff -Naur FlexGet-1.0r2558/flexget_vanilla.py FlexGet-1.0r2558.patch/flexget_vanilla.py
--- FlexGet-1.0r2558/flexget_vanilla.py	2010-10-17 19:36:51.000000000 +0300
+++ FlexGet-1.0r2558.patch/flexget_vanilla.py	2011-12-19 08:09:48.422991915 +0200
@@ -5,7 +5,7 @@
     You need to have all dependencies installed in site-packages when using this.
 """
 
-import flexget
+import flexget.ui
 
 if __name__ == '__main__':
-    flexget.main()
+    flexget.ui.main()
